src/hf_hydrodata/model/data_catalog_entry.csv

```diff
@@ -401,4 +401,5 @@ id,dataset,dataset_version,file_type,variable,dataset_var,entry_start_date,entry
 524,conus1_domain,,pfb,flow_direction,band_data,,,static,-,-,conus1,,1,/hydrodata/PFCLM/CONUS1_baseline/other_domain_files/flow_direction.pfb,"The flow directions for every grid cell (1=down, 2=left, 3=up, 4=right)",
 525,jasechko_2024,,sql,water_table_depth,wtd,01/01/1801,12/31/22,yearly,m,median,,,1,,,groundwater
 526,modis,,netcdf,snow_cover,snow_cover,3/10/2000,2/28/2017,daily,-,-,conus2,cy,2,/hydrodata/national_obs/modis/cloudfree_snow_cover/CONUS2/SnowUS_{cy}_CONUS2.nc,"This article presents a cloud-free snow cover dataset with a daily temporal resolution and 0.05 degree spatial resolution from March 2000 to February 2017 over the contiguous United States (CONUS). The dataset was developed by completely removing clouds from the original NASA's Moderate Resolution Imaging Spectroradiometer (MODIS) Snow Cover Area product (MOD10C1) through a series of spatiotemporal filters followed by the Variational Interpolation (VI) algorithm; the filters and VI algorithm were evaluated using bootstrapping test. The dataset was validated over the period with the Landsat 7 ETM+ snow cover maps in the Seattle, Minneapolis, Rocky Mountains, and Sierra Nevada regions. The resulting cloud-free snow cover captured accurately dynamic changes of snow throughout the period in terms of Probability of Detection (POD) and False Alarm Ratio (FAR) with average values of 0.955 and 0.179 for POD and FAR, respectively. The dataset provides continuous inputs of snow cover area for hydrologic studies for almost two decades. The VI algorithm can be applied in other regions given that a proper validation can be performed. It should be noted that there is a number of 12, 17, and 9 days missed in the years 2000, 2001, and 2002 respectively due to the unavailability in the original MODIS record.",
-527,conus2_domain,,pfb,ss_water_table_depth,wtd,,,static,-,-,conus2,,1,/hydrodata/PFCLM/CONUS2_baseline/steady_state_runs/2003_run/spinup.wy2003.wtd.pfb,steady state water table depth derived from steady state pressure and saturation,
\ No newline at end of file
+527,conus2_domain,,pfb,ss_water_table_depth,wtd,,,static,-,-,conus2,,1,/hydrodata/PFCLM/CONUS2_baseline/steady_state_runs/2003_run/spinup.wy2003.wtd.pfb,steady state water table depth derived from steady state pressure and saturation,
+528,fan_2013,,sql,water_table_depth,wtd,01/01/1927,12/31/2009,long_term,m,mean,,,1,,,groundwater

```

Comment:
I thought we opted for dates in the format `YYYY-mm-dd`

Reply:
You're right. I didn't know if it would mess things up to have a mix of `YYYY-mm-dd` and what is currently there for other files. It sounds like it won't, so I will update the format here.